acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/multikeysigner/MultiKeyAcceptanceTestBase.java

```diff
@@ -36,11 +39,12 @@
   @TempDir Path tomlDirectory;
 
   @AfterEach
-  public void cleanUp() {
+  public void cleanUp() throws IOException {
     if (ethSigner != null) {
       ethSigner.shutdown();
       ethSigner = null;
     }
+    MoreFiles.deleteDirectoryContents(tomlDirectory, RecursiveDeleteOption.ALLOW_INSECURE);
```

Comment:
why do we need to do this? using the @TempDir it should recursively delete the file contents